vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/profile.properties

```diff
@@ -1,7 +1,7 @@
 bdd.all-meta-filters=groovy: (!layout || layout.contains('desktop')) && !skip && (${bdd.meta-filters})
 
-selenium.grid.platform-name=Windows
-selenium.grid.platform-version=10
+selenium.grid.platform-name=
+selenium.grid.platform-version=
```

Comment:
should we reflect this in docs? since users relied on this implicitly

Reply:
I don't think so, nevertheless it makes sense to highlight it in 0.5.0 release docs